checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/model/CardBrand.kt

```diff
@@ -12,5 +12,5 @@ data class CardBrand(
 	val name: String,
 	val images: List<CardBrandImage>? = emptyList(),
 	val cvv: CardValidationRule? = null,
-	val pans: List<CardValidationRule>
+	val pans: CardValidationRule?  = null
```

Comment:
should this be a singular now that is no longer a list? Also none of these fields are ever going to be null except for cvv.

Reply:
updated to singular - also null pan pending discussion needed on whether we allow merchants to set their own CardBrands